jvm-pdf/src/main/groovy/org/asciidoctor/gradle/jvm/pdf/AsciidoctorPdfTask.groovy

```diff
@@ -49,15 +54,32 @@ class AsciidoctorPdfTask extends AbstractAsciidoctorTask {
         copyNoResources()
     }
 
-    /** The directory where custom are fonts to be found.
+    @Deprecated
+    File getFontsDir() {
+        if (this.fontDirs.size() > 1)
+            throw new Exception("There is more than 1 file in the fonts directory")
+        if (this.fontDirs.empty)
+            throw new Exception("No directory is specified")
+        this.project.file(this.fontDirs.first())
+    }
+
+    /** Specify a directory where to load custom fonts from.
      *
-     * @return Directory or {@code null} is no directory was set.
+     * This will set the {@code pdf-fontsdir} attribute
+     *
+     * @param f Directory where custom fonts can be found. anything convertible with {@link Project#file}
```

Comment:
Javadoc is going to need a `@deprecated` tag with some text to use `getFontsDirs` instead.